hudi-common/src/main/java/org/apache/hudi/common/table/log/HoodieLogFormat.java

```diff
@@ -146,26 +145,16 @@ class WriterBuilder {
     private String logWriteToken;
     // optional file suffix
     private String suffix;
-    // Rollover Log file write token
-    private String rolloverLogWriteToken;
```

Comment:
Is this removing because the `rolloverLogWriteToken` is always same as `logWriteToken`?

Reply:
yes